accounting/accounting.go

```diff
@@ -250,44 +252,45 @@ func (accounting *State) AddTransaction(txnr *idb.TxnRow) (err error) {
 		accounting.updateAccountType(stxn.Txn.Sender, string(ktype))
 	}
 
-	accounting.updateAlgo(stxn.Txn.Sender, -int64(stxn.Txn.Fee))
-	accounting.updateAlgo(accounting.feeAddr, int64(stxn.Txn.Fee))
+	accounting.updateAlgo(stxn.Txn.Sender, -int64(stxn.Txn.Fee.Raw))
+	accounting.updateAlgo(accounting.feeAddr, int64(stxn.Txn.Fee.Raw))
```

Comment:
I guess that this is a temporary until merging in the rest of the eval changes, right ?

Reply:
Yes, this code will be replaced.